### ISSUE TITLE
Add support for machine readable output

### ISSUE THREAD

**Issue by dgzlopes** [dgzlopes/tcp-latency#3]
Add a flag, where you can specify the desired format.

e.g `--output json` or `--output xml`

**Comment by Moras-del** [comment]
I can solve that.

**Comment by dgzlopes** [comment]
Nice @Moras-del! If you need help, feel free to ping me.

**Comment by Moras-del** [comment]
I would like to be sure. Do You want to support xml/json format only when program is used as module? E.g `def measure_latency(
    host,
    port = 443,
    timeout = 5,
    runs = 1,
    wait = 1,
    human_output = False,
    format='json'
) -> list:`
as I understand the content returned from the CLI should also be returned from the function call (when human_output=False, format='json/xml/None') in specified format, like:
`
'host': 'www.google.com',
'protocol': 'tcp',
'seq': '1',
'port': '443',
'timeout': '5',
'time': '200ms'
`
Am I correct?

**Comment by dgzlopes** [comment]
Yep. So we can use tools like [jq](https://stedolan.github.io/jq/) to work with the data. 

Human-output should be the default, but I think having other options (e.g JSON, XML, CSV, whatever) might be useful.

**Comment by Moras-del** [comment]
Ok but `measure_latency()` should also return list of json objects instead of latency points if specified? Or format should be used ONLY for working with CLI?

**Comment by dgzlopes** [comment]
In my opinion: CLI only :stuck_out_tongue:

I don't want to mess with how people invoke `measure_latency()`. And I don't think it's useful to get a list of JSON objects!

**Comment by Moras-del** [comment]
okay :)
